openqaoa/rqaoa/rqaoa_results.py

```diff
@@ -55,4 +70,63 @@ def get_hamiltonian_step(self, i):
         """
         Returns the Hamiltonian of the i-th step of the RQAOA.
         """
-        return self.get_problem_step(i).hamiltonian
\ No newline at end of file
+        return self.get_problem_step(i).hamiltonian
+
+    def dumps(self, human=False, string=False, indent=4):
```

Comment:
maybe we should change the `string` to something more explicative.

Maybe `format_as_string: Bool = False`

Reply:
for now I left `as_string`

Reply:
Sounds good! Marking it resolved